SparkPost/2web2ui
# hotfix/FE-958 - Fix layout problem with HealthScorePage component

Hotfix for FE-958

### What Changed
Made wrapping `Grid` only available in the OG theme.

### How To Test
- Sign in with `appteam` and verify the layout in Hibana on the route `/signals/health-score/sid/0` renders the 'Health Score Components' and 'Recommendations' laid out correctly
